### ISSUE TITLE
Test filesharing app

### ISSUE THREAD

**Issue by scottdear** [oceanbuilders/ob-iac#1]
Test and identify issues at https://files.oceanbuilders.com

**Comment by angiedear** [comment]
@diegopau tested my account, everything is great and working as expected.  @scottdear still needs to test his account.

**Comment by diegopau** [comment]
Thanks for testing! This is the error reported by @alessiodragoni :
"Just tried to create an account; after I received the email for activation, I had the attached error message when I opened the link."
![image](https://user-images.githubusercontent.com/315149/108980109-9fe44100-768b-11eb-99a5-ec358379359f.png)

@alessiodragoni do you mind trying again?

**Comment by scottdear** [comment]
@diegopau and @alessiodragoni There is an information disclosure security vulnerability.

Information disclosure, also known as information leakage, is when a website unintentionally reveals sensitive information to its users. Depending on the context, websites may leak all kinds of information to a potential attacker, including: Data about other users, such as usernames or financial information.

![image](https://user-images.githubusercontent.com/4229319/112761098-59e20c00-8fae-11eb-9aa6-4f50869bf66e.png)

In other words, it should not reveal if there is, or is not a particular e-mail or username in the system.  If they know an e-mail or username, they are halfway in.

Well maybe not half way, but at least 25% of the way.
